NavBar.html

```diff
@@ -0,0 +1,142 @@
+                // Example value provided here
+                geography: {
+                    state: {
+                        name: 'California'
+                    },
+                    county: {
+                        name: 'San Francisco'
+                    },
+                    zcta: {
+                        id: 94122
+                    }
+                },
+            }
+        },
+        /**
+         * The names of the parameter(s) for these "computed properties" correspond
+         * to attribute(s) in the "data" section (above).
+         * When one of these attributes changes, the property is recomputed.
+         */
+        computed: {
+            currentLevel: (crumbs) => (crumbs.length + 1)
+        },
+        methods: {
+            /**
+             * Pulls a list of breadcrumbs based on the geocode information
+             * contained within the given geography.
+             * @param {Object} geography
+             * @return {Array<string>}
+             */
+            getBreadcrumbsFromGeography = (geography) => {
+                if (!geography) return []
+                const crumbs = []
+                if (geography['state']) {
+                    const crumb = geography['state'].name
+                    crumbs.push(crumb)
+                }
+                if (geography['county']) {
+                    const crumb = geography['county'].name
+                    crumb += ' (county)'
+                    crumbs.push(crumb)
+                }
+                if (geography['city']) {
+                    const crumb = geography['city'].name
+                    crumb += ' (city)'
+                    crumbs.push(crumb)
+                }
+                if (geography['zcta']) {
+                    const crumb = geography['state'].id
+                    crumb += ' (zip code)'
+                    crumbs.push(crumb)
+                }
+                return crumbs
+            },
+            /**
+             * Opens the help modal
+             */
+            openHelpModal () {
+                this.get('helpModal').modal('open')
+            },
+            /**
+             * Navigates to the level corresponding to the clicked breadcrumb
+             */
+            goToLevel (level) {
+                const currentLevel = this.get('currentLevel')
+                if (level === currentLevel) return
+                // TODO: in a different PR, apply the level changes
+                // store.dispatch(...)
+            },
+        },
+        /**
+         * This function is called once, when the component is first created
+         */
+        oncreate () {
+            // Initialize the modal...
+            $(this.refs.helpModal).modal()
+            this.set({helpModal: $(this.refs.helpModal)})
+            // Initialize the breadcrumbs...
+            const crumbs = this.getBreadcrumbsFromGeography(
+                this.get('geography'))
+            this.set({crumbs})
```

Comment:
The formatting here is a little difficult to read. Can we refactor lines 137 & 138 so that the function is a one liner, such as this.getBreadcrumbsFromGeography(this.get('geography')) ?